Overview.src.html

```diff
@@ -688,25 +705,25 @@ <h4 id=terminology-headers>Headers</h4>
   <tr>
    <td>"<code title>fetch</code>"
    <td>""
-   <td>"<code title>subresource</code>"
+   <td>"<code title>fetch</code>"
    <td><code title>connect-src</code>
-   <td><code title=dom-global-fetch>fetch()</code>, Cache API?
+   <td><code title=dom-global-fetch>fetch()</code>, <code>XMLHttpRequest</code>, Cache API?
```

Comment:
Ah, I think I moved it while originally shuffling the values, and in my head fetch() and XHR are logically grouped together. We _can_ merge this row with the other connect-src one, modulo one thing: this row has its initiator set to "fetch" whereas the other is the empty string.. do we have anything depending on this?